### PR TITLE
Use one region for aws, eu-west-1

### PR DESCRIPTION
**Pull request by victorvarza**
Use one region, eu-west-1

### CODE REVIEW

File: src/aws/docker/main.tf [hunk 1]
```diff
@@ -98,9 +98,9 @@ resource "aws_security_group_rule" "allow_all_out" {
 
 module "dockerhost" {
   // Specify how many dockerhosts to deploy
-  count = 15
+  count = 1
   source          = "./dockerhost_module"
-  dockerhost-name = "cluster${count.index + 1}"
+  dockerhost-name = "dockerhost-${count.index + 1}"
```
Comment (adriananeci) [inline src/aws/docker/main.tf:103]: Cred ca e nevoie de `cluster` aici ca sa se coaguleze sub un singur director toate configurile cand facem export

File: src/aws/k8s-empty/main.tf [hunk 2]
```diff
@@ -5,21 +5,60 @@ provider "aws" {
...
-      allowed_cidr_blocks = "0.0.0.0/0",
-    },
-    cluster3 = {
-      allowed_cidr_blocks = "0.0.0.0/0",
-    },
-    cluster4 = {
-      allowed_cidr_blocks = "0.0.0.0/0",
-    },
-    cluster5 = {
+    cluster-empty1 = {
```
Comment (adriananeci) [inline src/aws/k8s-empty/main.tf:8]: cred ca si aici tre pastrat numele pt compatibilitate export configs. Tre validat.

**Review by adriananeci — CHANGES_REQUESTED** [review]

File: src/aws/docker/main.tf (hunk 1 above)
Comment (victorvarza) [reply src/aws/docker/main.tf:103]: e ok, am modificat scriptul de parsare si documentatia de setup: https://github.com/hlesey/k8s-labs-config/blob/main/src/export.py#L96

File: src/aws/k8s-empty/main.tf (hunk 2 above)
Comment (victorvarza) [reply src/aws/k8s-empty/main.tf:8]: la fel ca mai sus: https://github.com/hlesey/k8s-labs-config/blob/main/src/export.py#L96.

**Review by adriananeci — APPROVED** [review]

**Pull request merged**